bot/exts/evergreen/splitify.py

```diff
@@ -0,0 +1,208 @@
+        positioned accordingly.
+
+        8. Finally, it returns the new_img, the randomized squares (split pieces) stitched back into the format of the
+        original image - user's PFP.
+        """
+        random.shuffle(imgs)
+        single_img = imgs[0]
+
+        single_wdith = single_img.size[0]
+        single_height = single_img.size[1]
+
+        multiplier = int(math.sqrt(len(imgs)))
+
+        total_width = multiplier * single_wdith
+        total_height = multiplier * single_height
+
+        new_img = Image.new('RGBA', (total_width, total_height), (250, 250, 250))
+
+        width_multiplier = 0
+        height = 0
+
+        squares = math.sqrt(len(imgs))
+
+        for index, img in enumerate(imgs):
+            width = single_wdith * width_multiplier
+
+            new_img.paste(img, (width, height))
+
+            width_multiplier += 1
+
+            if (index + 1) % squares == 0:
+                width_multiplier = 0
+                height += single_height
+
+        return new_img
+
+    def splitify(self, img_bytes: BytesIO, squares: int) -> BytesIO:
+        """Seperate function run from an executor which splitifies an image."""
+        avatar = Image.open(BytesIO(img_bytes))
+        avatar = avatar.convert('RGBA').resize((1024, 1024))
+
+        img_squares = self.split_image(avatar, squares)
+        new_img = self.join_images(img_squares)
+
+        bufferedio = BytesIO()
+        new_img.save(bufferedio, format='PNG')
+        bufferedio.seek(0)
+
+        return bufferedio
+
+    @commands.command(name='splitify')
+    async def splitify_command(self, ctx: commands.Context, squares: int = 16) -> None:
+        """Splits your avatar into x squares, randomizes them and stitches them back into a new image!"""
+        async with ctx.typing():
+            if squares < 1:
+                raise commands.BadArgument('Squares must be a `positive number`:exclamation:')
+
+            if not math.sqrt(squares).is_integer():
+                raise commands.BadArgument('Squares must be a `perfect square`:exclamation:')
+
+            if squares > MAX_SQUARES:
+                raise commands.BadArgument('Number of squares cannot be higher than `193,600` :nerd:')
+
+            img_bytes = await ctx.author.avatar_url.read()
+            bufferedio = await self.bot.loop.run_in_executor(EXECUTOR, self.splitify, img_bytes, squares)
+
+            file = discord.File(bufferedio, filename='splitifed.png')
+
+            if squares == 1:
+                title = 'Hooh... that was a lot of work'
+                description = 'I present to you... Yourself!'
+            elif squares == MAX_SQUARES:
+                title = 'Testing the limits I see...'
+                description = 'What a masterpiece. :star:'
+            else:
+                title = 'Your splitified avatar'
+                description = 'Here is your avatar. I think it looks a bit *puzzly*'
```

Comment:
This is a bit weird gramatically.
```suggestion
                description = 'Here is your avatar. I think it looks a bit *puzzling*'
```